specs/create_Edit_pool.adoc

```diff
@@ -0,0 +1,113 @@
+// vim: tw=79
+
+= Create and Edit Pool
+
+Write a functionality for creating and editing a Pool in Tendrl-Dashboard UI
+
+
+== Problem description
+
+For Creating the new Pool we need to Provide forms with different fields
+* Pool Name
+* Number of Pools to create
+* Type()
+* Replicas
+* Cluster it belongs
+* Journal configration
+* OSD's
+* PG Count
+* Owner
+* Quotas
```

Comment:
Quotas values `quota_max_object` and `quota_max_bytes` need to be provided by end user, if quota enabled is selected.